# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../syslog-ng/head_n.patch
# Copyright (C) 2006 - 2011 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- syslog-ng-3.2.2/configure.in.orig	2011-01-15 17:14:31.000000000 +0100
+++ syslog-ng-3.2.2/configure.in	2011-04-26 17:28:52.236077819 +0200
@@ -262,7 +262,7 @@
 dnl ***************************************************************************
 
 AC_MSG_CHECKING(how to enable static linking for certain libraries)
-ldversion=`ld -V 2>&1 | head -1`
+ldversion=`ld -V 2>&1 | head -n 1`
 if echo $ldversion | egrep "GNU|Solaris" > /dev/null; then
         LD_START_STATIC="-Wl,-Bstatic"
         LD_END_STATIC="-Wl,-Bdynamic"
